From 241f77b0fe4aaaed0a7740c16a2cda52eea8479d Mon Sep 17 00:00:00 2001
From: Nina Gertskin <ngertskin@habana.ai>
Date: Wed, 27 Jan 2021 22:27:48 +0200
Subject: [PATCH] [SW-0] scripts: add dtc to path

mkimage script need to use BR2 dtc and not from /usr/bin/
so we add needed dtc to PATH

Change-Id: I186defddaf5bcf3328baaa77bcb803b2ad673a8f
Signed-off-by: Nina Gertskin <ngertskin@habana.ai>
---

diff --git a/board/habanalabs/gaudi/create-cpld-fit.sh b/board/habanalabs/gaudi/create-cpld-fit.sh
index a4c43cc..1c1081d 100755
--- a/board/habanalabs/gaudi/create-cpld-fit.sh
+++ b/board/habanalabs/gaudi/create-cpld-fit.sh
@@ -15,6 +15,9 @@
 CPLD_ITB_FILE=$IMAGE_DIRETORY/${HL_DEV}-cpld.itb
 KEYS_DIR=${BR2_EXTERNAL}/board/habanalabs/common/key/u-boot/
 KEY_FILE=hl_rsa3072.dtb
+
+export PATH=${HOST_DIR}/bin:$PATH
+
 # Set the security key index and version
 HL_SEC_FILE=$SPECS_EMBEDDED_ROOT/${HL_DEV}/${HL_DEV}_security.h
 HL_SEC_KEY_INDEX=`grep HL_SEC_KEY_INDEX $HL_SEC_FILE | sed 's/^.*\(0x.*\)$/\1/'`
diff --git a/board/habanalabs/gaudi/create-eeprom-fit.sh b/board/habanalabs/gaudi/create-eeprom-fit.sh
index 36258ba..612f9f4 100755
--- a/board/habanalabs/gaudi/create-eeprom-fit.sh
+++ b/board/habanalabs/gaudi/create-eeprom-fit.sh
@@ -16,6 +16,9 @@
 KEYS_DIR=${BR2_EXTERNAL}/board/habanalabs/common/key/u-boot/
 EEPROM_IMAGE_NAME=eeprom.txt
 EEPROM_IMAGE=${IMAGE_DIRETORY}/${EEPROM_IMAGE_NAME}
+
+export PATH=${HOST_DIR}/bin:$PATH
+
 # Set the security key index and version
 HL_SEC_FILE=$SPECS_EMBEDDED_ROOT/${HL_DEV}/${HL_DEV}_security.h
 HL_SEC_KEY_INDEX=`grep HL_SEC_KEY_INDEX $HL_SEC_FILE | sed 's/^.*\(0x.*\)$/\1/'`
diff --git a/board/habanalabs/gaudi/create-otp-image.sh b/board/habanalabs/gaudi/create-otp-image.sh
index 2c30859..d717c44 100755
--- a/board/habanalabs/gaudi/create-otp-image.sh
+++ b/board/habanalabs/gaudi/create-otp-image.sh
@@ -23,6 +23,7 @@
 OTP_ITS_FILE=${BR2_EXTERNAL}/board/habanalabs/common/${HL_DEV}-otp.its
 OTP_ITB_FILE=${IMAGE_DIRETORY}/${HL_DEV}-otp.itb
 
+export PATH=${HOST_DIR}/bin:$PATH
 source ${BR2_EXTERNAL}/board/habanalabs/common/image-utility.sh
 
 rm -f $OTP_IMAGE
diff --git a/board/habanalabs/gaudi/create-simple.sh b/board/habanalabs/gaudi/create-simple.sh
index 124747d..9350033 100755
--- a/board/habanalabs/gaudi/create-simple.sh
+++ b/board/habanalabs/gaudi/create-simple.sh
@@ -24,6 +24,8 @@
 SIMPLE_ITB_FILE=$IMAGE_DIRETORY/${HL_DEV}-${LOAD_TYPE_SUFFIX}.itb
 KEYS_DIR=${BR2_EXTERNAL}/board/habanalabs/common/key/u-boot/
 
+export PATH=${HOST_DIR}/bin:$PATH
+
 # Set the security key index and version
 HL_SEC_FILE=$SPECS_EMBEDDED_ROOT/${HL_DEV}/${HL_DEV}_security.h
 HL_SEC_KEY_INDEX=`grep HL_SEC_KEY_INDEX $HL_SEC_FILE | sed 's/^.*\(0x.*\)$/\1/'`
